pkg/components/exporter/exporter.go

```diff
@@ -175,6 +178,7 @@ func NewDaemonSet(detail components.Detail, k *v1alpha1.Kepler) *appsv1.DaemonSe
 						k8s.VolumeFromHost("proc", "/proc"),
 						k8s.VolumeFromHost("kernel-debug", "/sys/kernel/debug"),
 						k8s.VolumeFromHost("kernel-src", "/usr/src/kernels"),
+						k8s.VolumeFromSecret("redfish-cred", RedfishSecretName),
```

Comment:
On a related notes, there is a down side to this that  we should solve .. which is that secrets will be deleted when `kepler` is deleted :( ... Need to find a way to optionally delete ns if user created objects (including grafana exists).